microsoft-edge/devtools-guide-chromium/css/grid.md

```diff
@@ -26,30 +24,28 @@ CSS Grid is a powerful layout paradigm for the web.  A great place to get starte
 <!-- ====================================================================== -->
 ## Discover CSS grids
 
-When an HTML element on your page has `display: grid` or `display: inline-grid` applied to it, a `grid` badge is displayed next to it in the **Elements** tool:
+When an HTML element on your page has `display: grid` or `display: inline-grid` applied to it, a **grid** badge is displayed next to it in the **Elements** tool:
 
 ![Discover grid](./grid-images/grid-discover-grid.png)
 
-Click the badge to toggle the display of a grid overlay on the page.  The overlay appears over the element, laid out like a grid to display the position of the grid lines and tracks:
+Click the **grid** badge next to an element to display a grid overlay over that element in the rendered webpage.  The overlay appears over the element, and shows the position of the grid lines and tracks:
 
 ![Toggle grid badge](./grid-images/grid-highlight-grid.png)
 
-Open the **Layout** pane.  When grids are included on a page, the **Layout** pane includes a **Grid** section containing a number of options for viewing the grids.
+When grids are included on a page, the **Layout** pane in the **Elements** tool includes a **Grid** section that lists the grid containers and lets you configure the information which grid overlays should display.
 
 ![Layout pane](./grid-images/grid-layout-pane.png)
 
 The **Grid** section in the **Layout** pane contains the following 2 sub-sections:
 
-*  Overlay display settings
-*  Grid overlays
-
-<!--todo: verify the details for each of the sub-sections -->
+*  **Overlay display settings**, which lets you configure the information displayed in the grid overlay.
+*  **Grid overlays**, which lists the grid containers present on the webpage, and lets you enable or disable the corresponding grid overlays.
 
 
 <!-- ====================================================================== -->
 ## Overlay display settings
 
-In the **Layout** tab, in the expandable **Grid** section, the **Overlay display settings** section contains the following UI.
+In the **Layout** pane, in the expandable **Grid** section, the **Overlay display settings** section contains the following UI.
 
 ### Dropdown list
 
```

Comment:
maybe change col header from "Details" to "Purpose"
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/css/grid?branch=pr-en-us-3248#dropdown-list

Reply:
poor heading size usage, confusing - logical structure is not visible:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/css/grid?branch=pr-en-us-3248#dropdown-list
maybe demote subheadings:
**Hide line labels
Show line numbers
Show line names**

Reply:
add colons on lead-ins above pngs

Reply:
move image up to be after the lead-in
img link/filename:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/css/grid-images/grid-show-line-numbers.png

Reply:
move png up to follow sentence 1 in section:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/css/grid?branch=pr-en-us-3248#show-line-names

Reply:
in 
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/css/grid?branch=pr-en-us-3248#checkboxes
change col headings to: 
Checkbox
Purpose